packages/tools/src/commands/integrity/files.ts

```diff
@@ -77,32 +100,76 @@ const updateFactory = async (plugins: Workspace[]) => {
         const configKeyVar = `${camelCase}.CONFIG_KEY`;
 
         // Prepare content.
-        importContent += outdent`
-            import type { OptionsWith${pascalCase} } from '${plugin.name}/types';
-            import * as ${camelCase} from '${plugin.name}';
-        `;
-        typesExportContent += `export type { types as ${pascalCase}Types } from '${plugin.name}';`;
-        configContent += outdent`
-            if (options[${configKeyVar}] && options[${configKeyVar}].disabled !== true) {
-                plugins.push(...${camelCase}.getPlugins(options as OptionsWith${pascalCase}, context));
+        if (isInternalPluginWorkspace(plugin)) {
+            const getFunction = `get${pascalCase}Plugins`;
+            if (!pluginExports[getFunction]) {
+                errors.push(`[${error}] Missing ${green(getFunction)} in ${green(plugin.name)}.`);
+                continue;
             }
-        `;
 
-        // Only add helpers if they export them.
-        if (pluginExports.helpers && Object.keys(pluginExports.helpers).length) {
-            helperContent += `[${configKeyVar}]: ${camelCase}.helpers,`;
+            // Get the list of parameters to use when calling the internal plugin.
+            const paramsString = pluginExports[getFunction]
+                .toString()
+                // Find the list of arguments.
+                .match(/^\(([^)]+)\)/)[1]
```

Comment:
Is `pluginExports[getFunction]`always well formatted? If the argument list contains destructuring patterns or extra spacing / new lines it could mess up the regex matching and the subsequent split.